app/src/main/java/com/github/se/signify/model/navigation/Screen.kt

```diff
@@ -21,6 +21,7 @@ enum class Screen(val route: String, val requiresAuth: Boolean = true) {
   CREATE_CHALLENGE("CreateChallenge Screen"),
   CHALLENGE_HISTORY("ChallengeHistory Screen"),
   CHRONO_CHALLENGE("ChronoChallenge Screen/{challengeId}"),
+  TUTORIAL("Tutorial Screen"),
```

Comment:
> I fixed this in the LoginScreen.kt (line 152)

Upon testing, this is still broken. I think all you need to do is set this as not requiring authentication.

```suggestion
  TUTORIAL("Tutorial Screen", false),
```

Reply:
Okey I just saw that it was working before, without using the false argument. But I changed the logic so that the e2e that that we have pass. I think it's better if I address this issue in the next task.

Reply:
Alright, no problem.